src/game/common/gamemain.cpp

```diff
@@ -31,6 +33,8 @@ void Game_Main(int argc, char *argv[])
 {
     g_theGameEngine = Create_Game_Engine();
     g_theGameEngine->Init(argc, argv);
+    // TODO this should happen in GameEngine::Init but we don't have that yet
+    DISABLEDMASK_ALL.Set_All();
```

Comment:
ok so this was a void initDisabledMasks(void), theres also initKindOfMasks and initDamageTypeFlags in same location

Reply:
Added Init_Disabled_Masks. The others don't need implementing yet since we don't have KINDOFMASK_FS or DAMAGE_TYPE_FLAGS_ALL yet.